### PR TITLE
CON-2986 Test/Markdown ``DPxAI:Quest00/Ex05``

### PR DESCRIPTION
**Pull request by Oumaimafisaoui**
CON-2986

### CODE REVIEW

File: subjects/DPxAI/glance-on-power/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+## Glance on Power
+
+> Brainpower mode
+
+### Context
+
+Bravo! You've successfully brought the shape of your robot friend into existence. Now, it's time to take the next exciting step and start building the power to bring your robot to life. Yes, it's time to start thinking like a coder, builder! Using JavaScript, you'll be able to add dynamic abilities and interactions to your creation. Get ready to breathe life into your robot friend!
+
+> The platform includes a code editor, giving you an interface to write, test, and submit your exercises.
```
Comment (lotoussa) [inline subjects/DPxAI/glance-on-power/README.md:9]: this is more related to the beginning of the piscine imo

File: subjects/DPxAI/glance-on-power/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Ready to start collecting your power through learning to code? Let's start, time is against us!
```
Comment (lotoussa) [inline subjects/DPxAI/glance-on-power/README.md:11]: This is more related to the beginning of the quest/piscine imo

File: js/tests/glance-on-power_test.json
Comment (nprimo) [inline js/tests/glance-on-power_test.json:1]: the file should be named `glance-on-power.json` to be "recognized" by the `test.mjs`

**Review by nprimo — CHANGES_REQUESTED** [review]

File: js/tests/glance-on-power_test.json
Comment (lotoussa) [reply js/tests/glance-on-power_test.json:1]: ah yes thank you

**Review by lotoussa — APPROVED** [review]
lgtm

**Review by nprimo — APPROVED** [review]

**Pull request merged**